docs/source/new_project.rst

```diff
@@ -1,4 +1,4 @@
-Starting a new project
+Create a new templated project
```

Comment:
This page contains notes on each question in the template. The wording is slightly different between this and the `source/template_options` descriptions. We should choose one way of representing the questions.

Reply:
Yes - I am considering this out of scope of the "Prefactor" and consider it bundled in:

> The next step would be reviewing/revising content of pages.

You're right and it doesn't hurt to explicitly call it out as a to-do, though!